tests/resources/db2_test_tables.sql

```diff
@@ -32,42 +32,85 @@ CREATE TABLE dvt_core_types
 ,   col_string      VARCHAR(32000)
 ,   col_date        DATE
 ,   col_datetime    TIMESTAMP(3)
-,   col_tstz        TIMESTAMP(3)
+,   col_tstz        TIMESTAMP(3) -- Unable to create col_tstz with time zone on our test database therefore test data is adjusted.
```

Comment:
```
[db2inst1@db2server ~]$ db2 "CREATE TABLE db2inst1.dvt_date_timestamp(id INTEGER NOT NULL PRIMARY KEY, col_date DATE, col_datetime TIMESTAMP(3), col_tstz TIMESTAMP(3) WITH TIME ZONE);"
DB21034E  The command was processed as an SQL statement because it was not a 
valid Command Line Processor command.  During SQL processing it returned:
SQL0104N  An unexpected token "TIME" was found following "tz TIMESTAMP(3) 
WITH".  Expected tokens may include:  "REFERENCES".  SQLSTATE=42601
```
Adding here the output we get when trying to create a column with type TIMESTAMP WITH TIME ZONE since it seems it's not supported by DB2 LUW (image below), only by DB2 z/OS.

![image](https://github.com/user-attachments/assets/c6c54794-caeb-44f5-bc12-f421398aa53f)

References: 

- https://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.help.sqlanywhere.12.0.1/mlserver/ibm-datatypes-ml-ref.html,
- https://techdocs.broadcom.com/us/en/ca-mainframe-software/database-management/ca-fast-unload-for-db2-for-z-os/20-0/reference/keyword-descriptions/timestamp-with-time-zone-specify-the-data-format.html